v22.2/use-cloud-storage-for-bulk-operations.md

```diff
@@ -253,12 +257,61 @@ Has permission to assume role A. See [step 2](#step-2-user). | Has a trust polic
 
 When passing a chained role into `BACKUP`, it will follow this pattern:
 
+{% include_cached copy-clipboard.html %}
 ~~~sql
 BACKUP DATABASE movr INTO "s3://{bucket name}?AWS_ACCESS_KEY_ID={user's key}&AWS_SECRET_ACCESS_KEY={user's secret key}&ASSUME_ROLE={role A ARN},{role B ARN},{role C ARN}" AS OF SYSTEM TIME '-10s';
 ~~~
 
 Each chained role is listed separated by a `,` character. You can copy the ARN of the role from its summary page.
 
+### AWS workload identity
+
+With a CockroachDB cluster deployed on [Kubernetes](kubernetes-overview.html), you can allow your pods to authenticate as an IAM role that you have associated to a Kubernetes service account. You can then use assume role authentication to allow that IAM role to assume another role that has permissions to perform bulk operations to an S3 bucket.
+
+This means that a CockroachDB node will only be able to access credentials for the IAM role associated with the Kubernetes service account.
+
+You can use workload identities with assume role authentication to run the following operations:
+
+- [`BACKUP`](backup.html)
+- [`CREATE CHANGEFEED`](create-changefeed.html)
+- [`EXPORT`](export.html)
+- [`IMPORT`](import.html)/[`IMPORT INTO`](import-into.html)
+- [`RESTORE`](restore.html)
+
+#### Set up AWS workload identity
+
+First of all, it is necessary to create an IAM role for your Kubernetes service account to assume, and then configure your CockroachDB pods to use the service account. We will refer to this IAM role as an "identity role". You can complete all of these steps with Amazon's guide on [IAM roles for service accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html).
+
+Once you have an identity role that your CockroachDB nodes can assume, you can configure the identity role to assume another IAM role that contains the permissions to perform a bulk operation. 
```

Comment:
Can we label the bulk operation role as "operation role" like we have for GCP? The operation role is also referenced later on so it might be clearer to do so here.

Reply:
Agreed. Since we carefully describe the first paragraph as pertaining the the "identity role", when we switch to discussing the other one, here in the second paragraph, we definitely need to use the term: "operation role". Perhaps:

> Once you have an identity role that your CockroachDB nodes can assume, you will need to create an "operation role" containing the permissions to perform a bulk operation, which your identity role can assume.